src/internal/NeoEsp32I2sXMethod.h

```diff
@@ -129,7 +128,7 @@ class NeoEspI2sContext
         // construct only once on first time called
         if (I2sBuffer == nullptr)
         {
-            I2sBufferSize = DmaBytesPerPixelBytes * MaxBusDataSize;
+            I2sBufferSize = 32 * MaxBusDataSize * ((busNumber == 1 && BusMaxCount == 8) ? 1 : 2);
```

Comment:
add a comment to this change to evaluate for cleanup.  Its best to keep branching like this to a single location and wrap the concept into a const like the original.

Reply:
This was a cause of major memory violation. DmaBytesPerPixelBytes  was calculated in wrong way and it was way too small. It affected library stability and also the output signal was damaged and incompleted. I was thinking about how to encode it differently but I have no idea or they cause a reduction in readability: on the input 1 byte => 8 bits that should be multiplied by 4 bytes, not bits (x32) and additionally by two if I use 2 out of 4 bytes DMA (mode x8/x16 to is20)

Reply:
Your change is fine, The comment I request is so that I review in more depth after this merge to see if it can be cleaned up.

Reply:
OK, changed the code a bit and the comment was added